tests/Unit/Evolution/Systems/Cce/TestFunctions.py

```diff
@@ -0,0 +1,195 @@
+# Distributed under the MIT License.
```

Comment:
As of the guidelines introduced in #1382 please name the Python file the same as the cpp test file, without the `Test_` prefix.